hnn_core/tests/conftest.py

```diff
@@ -92,7 +92,9 @@ def _run_hnn_core_fixture(backend=None, n_procs=None, n_jobs=1,
                            't_evprox_2': 20,
                            'N_trials': 2})
             tstop = 40.
-        net = jones_2009_model(params, add_drives_from_params=True)
+        # Legacy mode necessary for exact dipole comparison test
+        net = jones_2009_model(params, add_drives_from_params=True,
+                               legacy_mode=True)
```

Comment:
I'm guessing the tests are failing because `legacy_mode=True` isn't getting passed to `_extract_drive_specs_from_hnn_params` in `_add_drives_from_params`?